maintainers/maintainer-list.nix

```diff
@@ -22865,6 +22865,13 @@
     githubId = 24979302;
     name = "Vladimír Zahradník";
   };
+  wgrav = {
+    name = "wgrav";
+    github = "wgrav01";
+    githubId = 63617338;
+    email = "wgrav01@protonmail.com";
+    keys = [ { fingerprint = "YkN3idqlpd+bvHQWVdrstvgm/3aNFRLrb6sa9i3HC+M"; } ];
```

Comment:
please use the base16 fingerprint of the GPG key that you use to sign your commits

Reply:
I use ssh to sign my commits, from my understanding I can't get a base16 fingerprint from that